src/cache_client.rs

```diff
@@ -167,41 +143,35 @@ impl CacheClient {
     /// * `key` - key of the item whose value we are setting
     /// * `value` - data to stored in the cache item
     ///
-    /// # Examples
+    /// # Optional Arguments
+    /// If you use [send_request](CacheClient::send_request) to set an item using a
+    /// [SetRequest], you can also provide the following optional arguments:
     ///
-    /// ```
-    /// # fn main() -> anyhow::Result<()> {
-    /// # use momento_test_util::create_doctest_cache_client;
-    /// # tokio_test::block_on(async {
-    /// # let (cache_client, cache_name) = create_doctest_cache_client();
-    /// cache_client.set(&cache_name, "k1", "v1").await?;
+    /// * `ttl` - The time-to-live for the item. If not provided, the client's default time-to-live is used.
```

Comment:
@cprice404 
All APIs with optional args except for `set` had an "Optional Arguments" section that I kept. Should I repeat the same verbiage on that last line? 

As for the APIs without additional optional arguments, I linked to the request objects anyway in case they want to send request objects directly but wasn't sure what else to mention there